### DISCUSSION TITLE
posts/2020/10/conversation-piece/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#5] [Announcements]
# posts/2020/10/conversation-piece/

Commenting options for your static website.

https://www.brycewray.com/posts/2020/10/conversation-piece/

**Comment by turystahobby** [comment]
I need some commenting system for web app about trucks truckercheckin.com

**Reply by brycewray** [reply]
This was about commenting for sites built with static site generators. However, from the code I see, it appears the site you mentioned is built with Joomla, so you might see if one of these extensions would suit you:

https://extensions.joomla.org/category/social-web/social-comments/
